src/memory/AmdFlash.cc

```diff
@@ -26,7 +26,7 @@ AmdFlash::AmdFlash(const Rom& rom, Chip chip_,
 	: motherBoard(config.getMotherBoard())
 	, chip(chip_)
 	, sectorInfo(sectorInfo_)
```

Comment:
If I read the code correctly, then `sectorInfo` is no longer needed after `init()`. Then maybe it can be passed as a parameter to `init()` and we don't need to store it as a member variable.

I think now `sectorInfo` only stores whether a sector is write protected or not. Maybe then we could give this parameter a more descriptive name?